app/Http/Controllers/StaffingController.php

```
@@ -142,63 +137,48 @@ protected function getGuildChannels($create = null)
      */
     public function store(Request $request)
     {
-        $this->validate($request, [
-            'event' => 'required|integer|exists:events,id',
-            'description' => 'required',
-            'channel_id' => 'required|integer',
-            'section_1_title' => 'required',
-            'section_2_title' => 'nullable',
-            'section_3_title' => 'nullable',
-            'section_4_title' => 'nullable',
+        $validated = $request->validate([
```

PHPMD (2.15.0)

Avoid unused local variables such as '$validated'. (undefined)

(UnusedLocalVariable)